features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/notifications/NotificationSettingsView.kt

```diff
@@ -69,7 +80,7 @@ fun NotificationSettingsView(
             NotificationSettingsState.MatrixSettings.Uninitialized -> return@PreferencePage
             is NotificationSettingsState.MatrixSettings.Valid -> NotificationSettingsContentView(
                 matrixSettings = state.matrixSettings,
```

Comment:
Should we avoid passing both `state.matrixSettings` and `state` separately here?

Reply:
I do that to avoid having to cast again `state.matrixSettings`. The type is `MatrixSettings` and here there is a smart cast to `MatrixSettings.Valid`.

Reply:
Ah, gotcha. I didn't notice the cast.